React/wbooks/src/app/components/Navbar/components/ProfileDropdown/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import { Redirect } from 'react-router';
+
+import profilePicture from '../../../../assets/profile_picture.png';
+import * as localStorageService from '../../../../../services/localstorageService';
+
+import styles from './styles.scss';
+
+class ProfileDropdown extends Component {
+  state = { isActive: false, isUserLogged: true };
+
+  handlePictureClick = event => {
+    this.setState({ isActive: !this.state.isActive });
+  }
+
+  handleSessionClick = event => {
+    localStorageService.removeUserAuthentication();
+    this.setState({ isUserLogged: false });
+  }
+
+  render() {
+    if(this.state.isUserLogged){
+      if(!this.state.isActive){
+        return (
+          <div className={styles.dropdownContainer}>
+            <img src={profilePicture} className={styles.profileIcon} alt='profileIcon' onClick={this.handlePictureClick}/>
+          </div>
+        );
+      }else {
+        return (
+          <div className={styles.dropdownContainer} >
+            <img src={profilePicture} className={styles.profileIcon} alt='profileIcon' onClick={this.handlePictureClick}/>
+            <div className={styles.buttonsContainer}>
+              <div className={styles.arrowContainer}>
+                <div className={styles.dropdownArrow}/>
+              </div>
+              <div className={styles.profileButton}>
+                <h1 className={styles.buttonText}>Perfil</h1>
+              </div>
+              <div className={styles.profileButton} onClick={this.handleSessionClick}>
+                <h1 className={styles.buttonText}>Cerrar sesión</h1>
+              </div>
+            </div>
+          </div>
+        );
+      }
+    }else {
+      return <Redirect to='login'/>
+    }
+  }
+}
+
+export default ProfileDropdown;
```

Comment:
Check that profile dropdown and notifications dropdown has the same container. What changes is each row. You are repeating all container styles in each dropdown component. i'd extract a `Dropdown` component that render a row children.

Reply:
It would be nice to have but this requires to change all components already made and that is gonna take more time. So as a conclusion i extracted all the repeated styles and codes to components ir order to improve this subject.
https://trello.com/c/Pg26ZbqN/34-logout-navbar